docs/howto/new-units.md

```diff
@@ -103,11 +102,6 @@ Here are the features.
     - **If omitted:** _this is usually fine to omit:_ most Units are only used with `Quantity`, not
       `QuantityPoint`.
 
-[^2]: Note that this requires us to build out multiplication and division between two
```

Comment:
Nit: Does this mean that the `[^3]` should be `[^2]`? It looks like it only needs to be unique, it gets rendered as 2 on the page.

Reply:
Nice catch!  I'd much rather keep the source code understandable, so I updated it.